pyanaconda/packaging/__init__.py

```diff
@@ -1080,6 +1080,31 @@ def _setupInstallDevice(self, storage, checkmount):
             url = method.url
             mirrorlist = method.mirrorlist
             sslverify = not (method.noverifyssl or flags.noverifyssl)
+        elif method.method == "hmc":
+            # Check if /dev/hmcdrv is already mounted.
+            if device == "/dev/hmcdrv":
+                log.debug("HMC is already mounted at %s.", DRACUT_REPODIR)
+                url = "file://" + DRACUT_REPODIR
+            else:
+                # Test the SE/HMC file access.
+                if iutil.execWithRedirect("/usr/sbin/lshmc", []):
+                    raise PayloadSetupError("The content of HMC media drive couldn't be accessed.")
+
+                # Test if a path is a mount point.
+                if os.path.ismount(INSTALL_TREE):
+                    log.debug("Don't mount the content of HMC media drive yet.")
```

Comment:
I think there should be more something like `Test if HMC is a mount point path.`. The actual message is without context what is happening, you just know what is not happening.

Reply:
Fixed.